haas/cli.py

```diff
@@ -247,17 +247,24 @@ def serve_networks():
     """Start the HaaS networking server"""
     from haas import model, deferred
     from time import sleep
+    import ConfigParser
     server.init()
     server.register_drivers()
     server.validate_state()
     model.init_db()
     migrations.check_db_schema()
+    # Check if config contains usable sleep_time
+    try:
+        sleep_time = float(cfg.get('devel', 'sleep_time'))
+    except (ConfigParser.NoSectionError, ConfigParser.NoOptionError,
+            ValueError):
+        sleep_time = 2
```

Comment:
My thought on this was that we wanted some sort of reasonable upper bound so that wrong things that could look like numbers (like a hex number) would get caught.

My suggestion is that in the example config file we just say "if set, must be > 0 and < 3600" to keep the documentation tidy. What do you guys think?